src/services/taskService.js

```diff
@@ -78,4 +79,29 @@ const editTask = async(accessToken,taskId, taskData) => {
         throw error;
     }
 }
-export {fetchTaskList, deleteTask, editTask}
+const createTask = async (accessToken, taskData) => {
```

Comment:
Please read the accessToken from this method from cookie directly instead of passing it in all Javascripts

Reply:
Fixed.

Reply:
Thanks. This looks fine...